sdk/netapp/Microsoft.Azure.Management.NetApp/src/Generated/IVolumesOperations.cs

```diff
@@ -192,7 +196,7 @@ public partial interface IVolumesOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string accountName, string poolName, string volumeName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string accountName, string poolName, string volumeName, bool? forceDelete = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
There are breaking changes in the interfaces. Here are 2 options:
- Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you can consider to bump the major version to `2.0.0`

Reply:
We bumped the major version, thanks

Reply:
NuGet doesn't really support semver like that. For track 2, we have a strict no-breaking changes policy and bumping the major version won't solve breaking changes. Instead, you can add an overloads. See https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-parameters for several tips/requirements how to do that. You can also hide the older one using `[EditorBrowsable(EditorBrowsableState.Never)]`.

Reply:
Thanks for the points, we will make the suggested changes

Reply:
customization added